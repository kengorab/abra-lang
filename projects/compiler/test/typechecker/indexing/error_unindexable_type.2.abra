val s = #{1, 2, 3}
s[0]