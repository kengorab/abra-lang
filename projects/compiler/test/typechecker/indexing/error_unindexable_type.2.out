Error at %FILE_NAME%:2:1
Unsupported indexing operation
  |  s[0]
     ^
Type 'Set<Int>' is not indexable
